experiments/temp.py

```diff
@@ -0,0 +1,80 @@
+import multiprocess as mp
```

Comment:
if we're going to keep this file might be worth changing it's name from `temp.py`, otherwise I would suggest removing if it's no longer needed.

Reply:
Oh yeah I meant to get rid of it